### PR TITLE
PQ-7 Add Alerts App on Provider Dashboard

### PR DESCRIPTION
**Pull request by slubwama**
https://issues.openmrs.org/browse/PQ-7

### CODE REVIEW

**Comment by dkayiwa** [comment]
Is the second commit for for a different ticket? I do not see the ticket id in the commit message.

**Comment by slubwama** [comment]
@dkayiwa Will Squash It

**Comment by dkayiwa** [comment]
Did you take a look at these merge conflicts?

**Comment by slubwama** [comment]
@dkayiwa done with resolving conflicts

File: omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package org.openmrs.module.patientqueueing.fragment.controller;
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java:10]: Can we put some license header?

File: omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.openmrs.ui.framework.fragment.FragmentModel;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AlertsFragmentController {
+	
+	protected final Log log = LogFactory.getLog(getClass());
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java]: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-loggingLogging

File: omod/src/main/webapp/resources/scripts/patientqueue.js [hunk 3]
```diff
@@ -3,6 +3,10 @@ var patientqueue = patientqueue || {};
 patientqueue.completePatientQueueDialog = null;
 patientqueue.patientId = null;
 patientqueue.sendPatientQueueDialog = null;
+patientqueue.readMessageDialog = null;
+patientqueue.alert_message_id=null;
```
Comment (dkayiwa) [inline omod/src/main/webapp/resources/scripts/patientqueue.js]: Can we put some space around equal signs?

**Comment by slubwama** [comment]
@dkayiwa  done with comments

**Comment by slubwama** [comment]
@dkayiwa done with reviews

File: omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AlertsFragmentController {
+
+	private final Logger log = LoggerFactory.getLogger(AlertsFragmentController.class);
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/patientqueueing/fragment/controller/AlertsFragmentController.java:29]: the above should be `static`

**Pull request merged**